packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
 
 ## Configuration
 
+### iconMenu
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
```

Comment:
There's some seemingly random extra whitespaces and a missing dot at the end here. I'd usually not care, but @Softwarecules just removed all extra whitespaces recently so the git change history will look more readable in the future (some of us have .md formatters that remove whitespaces automatically), hence I'm adressing it. :)

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0